_includes/v23.1/cdc/privilege-model.md

```diff
@@ -11,9 +11,23 @@ You can [grant](grant.html#grant-privileges-on-specific-tables-in-a-database) a
 GRANT CHANGEFEED ON TABLE example_table TO user;
 ~~~
 
-This privilege model provides a more granular way to grant users the ability to create a changefeed on a table. A user granted the `CHANGEFEED` privilege can create changefeeds on the target table even if the user does **not** have the [`CONTROLCHANGEFEED` role option](alter-role.html#role-options) or the `SELECT` privilege on the table. 
+When you grant a user the `CHANGEFEED` privilege, they can:
 
-Since you can grant the `CHANGEFEED` privilege to a user or role **without** them needing the `SELECT` privilege on a table, these users will be able to create changefeeds, but they will not be able to run a `SELECT` query on that data directly. {% if page.name == "create-changefeed.md" %} However, these users could still read this data indirectly if they have read access to the [sink](changefeed-sinks.html), or create a "sinkless" changefeed that emits messages to the SQL session.{% else %} However, these users will be able to read this data indirectly as `EXPERIMENTAL CHANGEFEED FOR` emits changefeed messages to the SQL session. {% endif %}
+- Create changefeeds on the target table even if the user does **not** have the [`CONTROLCHANGEFEED` role option](alter-role.html#role-options) or the `SELECT` privilege on the table. 
+- {% include_cached new-in.html version="v23.1" %} Manage the changefeed jobs that they have created using the [`SHOW CHANGEFEED JOB`](show-jobs.html), [`PAUSE JOB`](pause-job.html), [`RESUME JOB`](resume-job.html), and [`CANCEL JOB`](cancel-job.html) commands.
+
+These users will be able to create changefeeds, but they will not be able to run a `SELECT` query on that data directly. However, they could still read this data indirectly if they have read access to the [sink](changefeed-sinks.html).
+
+{% include {{ page.version.version }}/cdc/ext-conn-cluster-setting.md %}
+
+### Privilege model
+
+Granted privileges | Usage 
+-------------------+-------
+`CHANGEFEED` | Create changefeeds on tables defined in the `GRANT` statement.<br>Manage the changefeed job.
+`CHANGEFEED` + `USAGE` on external connections | Create changefeeds on tables defined in the `GRANT` statement **only** to an external connection URI that the user has [`USAGE`](create-external-connection.html#required-privileges) on.<br>Manage the changefeed job.
+`SELECT` | Create a sinkless changefeed that emits messages to a SQL client.
+**Deprecated** `CONTROLCHANGEFEED` role option + `SELECT` | Create changefeeds on tables defined in the `GRANT` statement.
```

Comment:
The 2nd row is a bit confusing because it's only applicable when `changefeed.permissions.enforce_external_connections=true`. Consider adding that cluster setting to the left column of that row. Or, you can move the `{% include {{ page.version.version }}/cdc/ext-conn-cluster-setting.md %}` under this table here and put that row in its own table right underneath the note.

Also consider putting the target in the left column. Ex. CHANGEFEED on tables + USAGE on ...
Ex. CONTROLCHANGEFEED role option + SELECT on tables. Then in the right column, you can just say "Create changefeeds on tables". I'm not sure about including "defined in the `GRANT` statement" just because a user could be granted privileges in other ways such as multiple grant statements, by role inheritance, or by an import/migration.

Reply:
@jayshrivastava For the `CHANGEFEED` + `USAGE` on an external connection.... the cluster setting `changefeed.permissions.enforce_external_connections=true` is **enforcing**  that: 

> "When you enable this setting, users with the `CHANGEFEED` privilege on a set of tables can only create changefeeds into [external connections]"... correct? Or no?

So, what about if a cluster admin doesn't need to enforce users use external connections, rather, they just want a changefeed to emit to a particular external connection (sink), wouldn't a user just need the `CHANGEFEED` + `USAGE` (on the ext. conn.)? 

That is, there is a scenario where external connection usage is enforced for changefeeds across a cluster. But, there is another scenario where an admin/`externalconnection` user just wants to grant `USAGE` on a particular ext. connection so that a user can run a changefeed to it? (Without flipping the cluster setting to `true`.) 

If that is the case, perhaps there needs to be a row added for both:

`CHANGEFEED` + `USAGE` on external conn
 `CHANGEFEED` + `USAGE` on external conn + cluster setting

If you confirm/un-confirm that, I will take another shot at the usage component of this table.

Reply:
That's a good point. You are correct. I think it makes sense to leave that row in the table. Even if the setting is `false`, a user will needs `CHANGEFEED` on the tables + `USAGE` on the external conn to create a changefeed into the external conn.

Reply:
I reworked the table a little. Mainly:

- Added in a note around the changefeed + ext conn row
- Added a sentence before the table so it's clear the user has privileges on the tables we mention in the table (so many tables!)
- Adjusted the language in the usage column as you mention